README.org

```diff
@@ -234,17 +236,36 @@ var detail = {
     [uniqueId | 'id']: String, // the unique id
     title: String,             // title of the asset
     year: Number,              // year of release
-    genre: [movie.Genre],      // a free list of genre keywords
+    genres: [String],          // a free list of genre keywords
     rating: Number,            // a 0-100 rating value
-    poster: String,            // url to poster image
+    poster: String,            // poster image
```

Comment:
i think we should keep it being a url, before  type was url, maybe it's better ?